### PR TITLE
Feature/plp new arrival

### PR DESCRIPTION
**Pull request by Deepesh-emproto**

### CODE REVIEW

File: src/components/CommonCourselPDP/CommonCourselPDP.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        <span />
+          <h2>{text}</h2>
+        <span />
+      </div>
+      <div>
+      <Slider {...settings} className={styles.commonCourselSlider}>
+      {
+        images?.map((val)=>{
+          return(
+            <CardComponent key={Math.random()} height={'40%'} width={'80%'} images={val.image} />
```
Comment (prateekoli9) [inline src/components/CommonCourselPDP/CommonCourselPDP.tsx:60]: Don't use Math.random as key anywhere

File: src/backend/PDPCourselData.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import pdpImage from '../Assets/jayporeImage.jpg';
```
Comment (prateekoli9) [inline src/backend/PDPCourselData.tsx:1]: Remove this file, add this in function inside productDescription.ts file in backend folder

File: src/backend/collection.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import pdpImage from '../Assets/plpPageImage.webp';
```
Comment (prateekoli9) [inline src/backend/collection.ts:1]: Same comment as above

File: src/backend/home.ts [hunk 4]
```diff
@@ -120,7 +120,7 @@ export const Obj: Name = {
                         <p style=margin-left:1rem>Flat 15% Off site wide till 5th September, 2022.</p> 
                         <p >*T&amp;C Apply</p>
                       </span> &nbsp;
-                      <span class="msgClose" style="display: block;"></span> 
+                      MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-o69gx8-MuiCardMedia-root   <span class="msgClose" style="display: block;"></span>
```
Comment (prateekoli9) [inline src/backend/home.ts:123]: Remove if not needed

**Comment by Deepesh-emproto** [comment]
Screenshot from 2022-10-03 17-33-51

File: src/App.tsx [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import RouterComponent from '@components/router/router.component';
 import Footer from '@components/Footer/Footer.component';
 import OfferBanner from '@components/Header/OfferBanner.component';
-import HeaderComponent from '@components/Header/Header.component';
+import HeaderComponent from './components/header/header.component';
```
Comment (prateekoli9) [inline src/App.tsx:4]: Revert this changes

File: src/components/CommonCourselPDP/CommonCourselPDP.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
+import { FC } from 'react';
```
Comment (prateekoli9) [inline src/components/CommonCourselPDP/CommonCourselPDP.tsx:1]: Make a common productCard instead of CommonCarousel

File: src/components/NewArrival/NewArrival.component.tsx [hunk 7]
```diff
@@ -0,0 +1,19 @@
+import {useState,useEffect} from 'react';
```
Comment (prateekoli9) [inline src/components/NewArrival/NewArrival.component.tsx:1]: Rename component to NewArrivals

File: src/components/CollectionPDP/CollectionPDP.tsx [hunk 8]
```diff
@@ -0,0 +1,38 @@
+import {useState,useEffect} from 'react';
```
Comment (prateekoli9) [inline src/components/CollectionPDP/CollectionPDP.tsx:1]: Rename component to MoreInCollection.

**Comment by Deepesh-emproto** [comment]
Screenshot from 2022-10-06 18-04-28


MORE FROM THIS COLLECTION

**Comment by Deepesh-emproto** [comment]
Screenshot from 2022-10-06 18-07-16


Responsive View

File: src/App.tsx (hunk 5 above)
Comment (prateekoli1) [inline src/App.tsx:4]: Revert this change

**Review by prateekoli1 — APPROVED** [review]
